.github/workflows/daily-build.yml

```
@@ -57,6 +57,21 @@ jobs:
           echo '$build_number='$build_number
           echo "BUILD_NUMBER=$build_number" >> $GITHUB_ENV
 
+      - name: 'Save workflow parameters'
+        run: |
+          mkdir -p ./workflow-data
+          echo "WORKFLOW_GITHUB_SHA=${{ github.sha }}" >> ./workflow-data/params.env
+          echo "WORKFLOW_GITHUB_REF_NAME=${{ github.ref_name }}" >> ./workflow-data/params.env
+          echo "WORKFLOW_BUILD_NUMBER=${{ env.BUILD_NUMBER }}" >> ./workflow-data/params.env
+      
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)